components/core/src/clp_s/CommandLineArguments.hpp

```diff
@@ -80,8 +88,54 @@ class CommandLineArguments {
         return m_metadata_db_config;
     }
 
+    std::string const& get_reducer_host() const { return m_reducer_host; }
+
+    int get_reducer_port() const { return m_reducer_port; }
+
+    reducer::job_id_t get_job_id() const { return m_job_id; }
+
+    bool do_count_results_aggregation() const { return m_do_count_results_aggregation; }
+
+    OutputHandlerType get_output_handler_type() const { return m_output_handler_type; }
+
 private:
     // Methods
+    /**
```

Comment:
Should we provide descriptions for these methods, or is it redundant given that their names already convey their purpose effectively?

Reply:
I can add a short description for each of these.